kqueen_ui/blueprints/ui/templates/ui/login.html

```diff
@@ -39,7 +39,7 @@ <h3 style="margin: 5px 0px;">Log In</h3>
   </div>
   <div class="panel panel-default text-center">
     <div class="panel-body" style="font-size: 0.9em;">
-      {% if config.ENABLE_PUBLIC_REGISTRATION %}
+      {% if config.ENABLE_PUBLIC_REGISTRATION == 'True' %}
```

Comment:
I would prefer if you would also change the default values:
- https://github.com/Mirantis/kqueen-ui/blob/master/kqueen_ui/config/base.py#L25 - from `False` to `'False'` 
- https://github.com/Mirantis/kqueen-ui/blob/master/kqueen_ui/config/prod.py#L20 - from `True` to `'True'`
so we don't mix variable types if we are comparing strings now

Reply:
sure, done